psl/builtin-connectors/src/mysql_datamodel_connector.rs

```diff
@@ -87,7 +87,7 @@ impl Connector for MySqlDatamodelConnector {
     }
 
     fn is_provider(&self, name: &str) -> bool {
-        name == "mysql" || name == "@prisma/mysql"
+        name == "mysql"
```

Comment:
This change broke support for `prisma migrate`. Before, you could `db pull` from a schema such as:

```prisma
datasource db {
  provider = "@prisma/mysql"
  url      = env("TEST_DATABASE_URL")
}
```

This would use the Rust drivers for pulling/pushing table definitions, and the given JS connector for submitting queries via Prisma Client.

Now, with this change, users are forced to manually replace the `datasource.provider` entry when they want to pull / push data:

```diff
datasource db {
-   provider = "@prisma/mysql"
+   provider = "mysql"
  url      = env("TEST_DATABASE_URL")
}
```

worsening the DX.

This is particularly undesired, if we consider that using JS connectors for `prisma migrate` is outside of the scope of #project-node-drivers.